packages/components/src/components/common/table/table.css.ts

```diff
@@ -36,15 +35,12 @@ export const footer = style({
   backgroundColor: vars.color.n100
 });
 
-export const row = style([
-  transitionColors,
```

Comment:
removed also the transition colors on rows (so the select color has an animation, see gifs).
Because it also has strange side effects, e.g. on trees and IMO not that important :)